src/components/organisms/AdminVenueView/components/RoomCard/RoomCard.tsx

```diff
@@ -101,12 +101,14 @@ export const RoomCard: React.FC<RoomCardProps> = ({
             iconName={room.isEnabled ? faEye : faEyeSlash}
             disabled={isTogglingRoom}
             onClick={toggleRoom}
+            title={`click to ${room.isEnabled ? "hide" : "show"} room`}
```

Comment:
The prop is `title` but the visual feedback is a tooltip. I suppose you are using the default functionality that displays an elements title as tooltip?

Reply:
Default for now. Can update if new design provided.